lib/internals/src/utils/action/action-name-creator.ts

```diff
@@ -1,14 +1,14 @@
 import { MethodArgsRegistry } from '../method-args-registry/method-args-registry';
 
 interface ActionNameCreatorOptions {
-    stateName: string | null;
+    statePath: string;
     methodName: string;
     argumentsNames: string[];
     argumentRegistry?: MethodArgsRegistry;
 }
 
 export function actionNameCreator(options: ActionNameCreatorOptions): string {
-    const { stateName, argumentsNames, methodName, argumentRegistry }: ActionNameCreatorOptions = options;
+    const { statePath, argumentsNames, methodName, argumentRegistry }: ActionNameCreatorOptions = options;
```

Comment:
not very good to combine two PRs in one

Reply:
Absolutely right, sorry about that. Can we close this PR and I'll make a new one with your proposed solution and without changes from my other PR?